### PR TITLE
feat: Add github link button to learn documentation pages

### PR DESCRIPTION
**Pull request by miedziak**

### CODE REVIEW

File: scripts/get-depended-repos.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+const neatCsv = require('neat-csv');
+
+const RepoParser = require('../lib/RepoParser');
+
+const getDependedRepos = async () => {
+  return new Promise((resolve, reject) => {
+    fs.readFile('./dependencies_repos.csv', async (err, data) => {
```
Comment (Necromos) [inline scripts/get-depended-repos.js]: **T** This is going to be a Buffer. As it's not a big problem but you could easily use https://nodejs.org/api/fs.html#fs_fs_createreadstream_path_options so there won't be need for this callback :)

Comment (miedziak) [reply scripts/get-depended-repos.js]: @Necromos Would't I need to pass callback for this `on` method in read stream ?
```
    readStream.on('data', chunk => {
        // do something with chunk
    });
```

File: src/components/EditOnGithubLink.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import Button from "./Button";
+
+const Container = styled('div')`
+  padding: 3.5rem 0;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+`;
+
+const findMatchInDependedRepos = (filePath, infos) => {
```
Comment (LWronka) [inline src/components/EditOnGithubLink.js]: May be better name could be `findFilePathInDependedRepos ` or `findPathInDependedRepos `

File: src/components/EditOnGithubLink.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+const generateDependedGithubUrl = (repo, filePath) => {
+  const slugPathLength = repo.slug.split('/').length;
+  const githubFilePath = filePath.split("/").slice(slugPathLength).join("/");
+  return `${repo.githubUrl}/${githubFilePath}`;
+};
+
+const EditOnGithubLink = ({ filePath, dependedRepos}) => {
+  const {ockamWebsiteRepo, markdownPath, githubProductionPath} = useSiteMetadata();
+  const repo = findMatchInDependedRepos(filePath, dependedRepos);
```
Comment (LWronka) [inline src/components/EditOnGithubLink.js]: `repo` is a boolean value so may be could be added `is` to the name or sth like that to tell developer about it ?

File: src/theme/defaultTheme.js [hunk 4]
```diff
@@ -15,6 +15,7 @@ const colors = {
   text: '#A2B6C7',
   code: '#e7e8e8',
   primary: '#52c7ea',
+  primaryBleak: '#183643',
   dirtyPrimary: '#3C8DA9',
```
Comment (LWronka) [inline src/theme/defaultTheme.js]: May be first should be adjective then main color ? now is mixed:
  bleakPrimary: '#183643',
  dirtyPrimary: '#3C8DA9',

File: src/components/EditOnGithubLink.js (hunk 3 above)
Comment (miedziak) [reply src/components/EditOnGithubLink.js]: Not actually, function `findMatchInDependedRepos` returns `object | undefined`. There find method there.

**Comment by mrinalwadhwa** [comment]
merged https://github.com/ockam-network/website/commit/5cc38bfb7160fd53adfca0e30f448d651b995971

**Pull request closed without merging**